subgraph-js/src/schema-helper/buildSchemaFromSDL.ts

```diff
@@ -221,6 +222,7 @@ export function buildSchemaFromSDL(
       schemaDirectives.push(
         ...(definition.directives ? definition.directives : [])
       );
+      description = definition.description?.value
```

Comment:
Can we capture the whole node here (`definition.description`) and use it below? It should even properly be a `StringValueNode | undefined`, which is what `astNode.description` is expecting.

The only uncertainty I have with this suggestion is whether or not it's important that you're setting `block: true` down below. Seems like it would be fine to allow `block` to be whatever it was originally on the node?